platforms/android/coverage.gradle

```diff
@@ -1,37 +1,29 @@
-def excludes = [
-// dependency injection graph
-'**/*Module.*',
-'**/*Module*.*',
-
-// Framework entry points
-'**/*Activity*',
-'**/*Fragment*',
-'**/*Application*',
-'**/*AndroidService*',
```

Comment:
This config had been copied across from Element Android but it's not needed in this code base